app/javascript/components/pages/PitchCardShow.test.js

```diff
@@ -15,9 +15,10 @@ describe("When PitchCardShow renders", () => {
     pitch:"test"
   }
   it("displays a pitch card", () => {
-    const wrapper = shallow(<PitchCardShow pitchCard={pitchCard}/>)
-    const test = wrapper.find("#header").text()
-    expect(test).toEqual("Pitch Card Show")
+    // const wrapper = shallow(<PitchCardShow pitchCard={pitchCard}/>)
+    // const test = wrapper.find("#header").text()
+    // expect(test).toEqual("Pitch Card Show")
+    // !----Commented out above since Header is removed
```

Comment:
Let's get rid of this code then if there is no header

Reply:
just removed, thank you!